RHEL/7/input/xccdf/system/software/gnome.xml

```diff
@@ -12,7 +12,72 @@ Red Hat Graphical environment.
 For more information on GNOME and the GNOME Project, see <b>https://www.gnome.org</b>
 </description>
 
-<Rule id="dconf_gnome_disable_user_list">
+<Group id="gnome_login_screen">
+<title>Configure GNOME Login Screen</title>
+<description>In the default GNOME3 desktop, the login is displayed after system boot
+and can display user accounts, allow users to reboot the system, and allow users to
+login automatically and/or with a guest account. The login screen should be configured
+to prevent such behavior.
+<br /><br />
+For more information about enforcing preferences in the GNOME3 environment using the DConf
+configuration system, see <b>http://wiki.gnome.org/dconf</b> and
+the man page <tt>dconf(1)</tt>. For Red Hat specific information on configuring DConf
+settings, see <b>https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Desktop_Migration_and_Administration_Guide/index.html</b>
+</description>
```

Comment:
I'm super leery of pointing to gnome.org. The upstream may have significantly different versions and config options than shipping in RHEL.  The other pointers make absolute sense!